src/qt/guiutil_font.cpp

```diff
@@ -35,8 +35,8 @@ static int fontScale = g_font_defaults.scale;
 // Contains the weight settings separated for all available fonts
 static std::map<FontFamily, std::pair<QFont::Weight, QFont::Weight>> mapDefaultWeights;
 static std::map<FontFamily, std::pair<QFont::Weight, QFont::Weight>> mapWeights;
-// Contains all widgets and its font attributes (weight, italic, size) with font changes due to GUIUtil::setFont
-static std::map<QPointer<QWidget>, std::tuple<FontWeight, bool, int>> mapFontUpdates;
+// Contains all widgets and its font attributes (family, weight, italic, size) with font changes due to GUIUtil::setFont
+static std::map<QPointer<QWidget>, std::tuple<FontFamily, FontWeight, bool, int>> mapFontUpdates;
```

Comment:
consider to use here a new struct.

4 params (family, weight, italic, size) it's too many.
std::tuple is a good choice if data is uniform or auto-generated (or util/template code somewhere), but `struct` here is a good fit

Reply:
Resolved in latest push